### PR TITLE
Document the --progress/-p options for system:cron

### LINKED ISSUE #1955: new option --progress for occ system:cron
see https://github.com/owncloud/core/pull/36304

The newly added option --progress/-p will show the progress bar. Without there will be no outpur

### PR DESCRIPTION
**Pull request by settermjd**
This fixes #1955 by documenting the `--progress` and `-p` options available to `system:cron` in v10.3.1. It also adds in help output, so that the command is fully understood.

**Only** backport to 10.3 - **not to any earlier branch**.

### CODE REVIEW

**Comment by mmattel** [comment]
Disagree with the changes because it breakes the look and feel compared to the current occ command set. I have highlighted in #1955 how to fix that easily (copy / paste + content adoption).

**Comment by settermjd** [comment]
@mmattel, have revised that section to keep it inline with the rest of the document.

File: modules/admin_manual/pages/_partials/configuration/server/occ_command/system_command.adoc [hunk 1]
```diff
@@ -6,13 +6,66 @@
...
+
+== Options
+
+[width="100%",cols="20%,70%",]
+|===
+| -p, --progress        
+| Shows a progress bar - for use in manual execution. 
+Do not use when executing from crontab
+
+| -h, --help
```
Comment (mmattel) [inline modules/admin_manual/pages/_partials/configuration/server/occ_command/system_command.adoc]: You can remove all standard options like -h, -q ect as they are described generally on top and are valid for all commands. Only the particular options/arguments need to be described.

Comment (settermjd) [reply modules/admin_manual/pages/_partials/configuration/server/occ_command/system_command.adoc]: Sounds fair.

**Comment by settermjd** [comment]
@mmattel, common command options have been removed.

**Pull request merged**